### DISCUSSION TITLE
Telemetry viewer

### DISCUSSION THREAD

**Discussion by xsampedro** [br-g/openf1#14] [Show and tell]
Huge shoutout for providing the API! 👏

Currently developing a telemetry viewer, still in progress. Check it out here: https://xsampedro.com/openf1/

**Comment by br-g** [comment]
Thanks for sharing this! 
Your app looks great already. Looking forward to see how the project evolves! 🚀

**Comment by xsampedro** [comment]
For anyone interested, I have made the repository public: https://github.com/xsampedro/f1_telemetry_viewer
